manuals/manual_convert_schedule_to_events.md

```diff
@@ -0,0 +1,463 @@
+<h3 style="color: #f46242;"><strong>Warning</strong>: <i>the specified events below are not yet implemented within Pronto yet and thus cannot be processed</i></h3>
```

Comment:
I would remove this. We can easily add & ignore them should any suppliers be very fast (which I doubt)